frontend/packages/dev-console/src/components/pipelines/modals/triggers/TriggerBindingSection.tsx

```diff
@@ -27,16 +30,23 @@ const TriggerBindingSection: React.FC = () => {
         />
         {bindingVars && (
           <Expandable toggleTextExpanded="Hide Variables" toggleTextCollapsed="Show Variables">
-            <div className="odc-add-trigger-form__variable-container">
-              <p className="odc-add-trigger-form__variable-descriptor">
+            <div className="odc-trigger-binding-section__variable-container">
+              <p className="odc-trigger-binding-section__variable-descriptor">
                 The following variables can be used in the Parameters or when created new Resources.
               </p>
               {bindingVars.map(({ name }) => (
-                <Badge key={name} className="odc-add-trigger-form__variable-badge" isRead>
+                <Badge key={name} className="odc-trigger-binding-section__variable-badge" isRead>
                   {name}
                 </Badge>
               ))}
             </div>
+            <FormHelperText
+              isHidden={false}
+              className="odc-trigger-binding-section__variable-help-text"
+            >
+              Use this format when referencing variables in this form:{' '}
+              <code>{'${params.parameter}'}</code>
```

Comment:
```suggestion
              <code>{'$(params.parameter)'}</code>
```
Did the design have `{`? If so, I missed that. Pipelines use `$()`

Reply:
Ah no, it was `(` 😄

Reply:
So the design doesn't use a code block... but that is an interesting idea. This change is I think more inline with what they were asking for.
```suggestion
              &lsquo;$(params.parameter)&rsquo;
```

It's a temporary note to the user while we work on the UX of the situation and make a better system to just select / autocomplete variable names.

Reply:
Personally, I like the usage with `<code>` block, it looks good and is semantically correct but i would let UX team decide that. I'll change it once we have an answer from them.